src/components/Reviews.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import { getReviews } from "../api";
+
+const Reviews = () => {
+  const [reviewsList, setReviewsList] = useState([
+    {
+      title: "One Night Ultimate Werewolf",
+      designer: "Akihisa Okui",
+      owner: "happyamy2016",
+      review_img_url:
+        "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+      category: "hidden-roles",
+      created_at: 1610964101251,
+      votes: 5,
+    },
+  ]);
```

Comment:
We should probably have this state as an empty Array to begin with, otherwise it creates the illusion that we have data to begin with when we do not. If you are looking to have something before we do the fetch, then a loading state may be more appropriate in this case.